### PR TITLE
Upgrade to docker-java@3.2.12

### PR DESCRIPTION
**Pull request by james-toussaint**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -54,11 +54,15 @@ dependencies {
...
-    implementation 'com.github.docker-java:docker-java-transport-httpclient5:3.2.5'
+    implementation 'com.github.docker-java:docker-java:3.2.12'
+    // Using zerodep until docker client stops blocking after having called
+    // listContainersCmd() over httpclient5
+    implementation group: 'com.github.docker-java', name: 'docker-java-transport-zerodep', version: '3.2.12'
 
     // failsafe
     implementation 'net.jodah:failsafe:2.4.4'
+    //TODO Migrate from Awaitility to Failsafe everywhere
+    implementation group: 'org.awaitility', name: 'awaitility', version: '4.1.1'
```
Comment (jbern0rd) [inline build.gradle]: I don't see where this dependency is being used.
Why has it been added here ?

File: build.gradle [hunk 2]
```diff
@@ -54,11 +54,15 @@ dependencies {
     implementation 'io.github.openfeign:feign-slf4j:11.0'
 
     // docker
-    implementation 'com.github.docker-java:docker-java:3.2.5'
-    implementation 'com.github.docker-java:docker-java-transport-httpclient5:3.2.5'
+    implementation 'com.github.docker-java:docker-java:3.2.12'
+    // Using zerodep until docker client stops blocking after having called
+    // listContainersCmd() over httpclient5
+    implementation group: 'com.github.docker-java', name: 'docker-java-transport-zerodep', version: '3.2.12'
```
Comment (jbern0rd) [inline build.gradle]: Shouldn't we stick to a single dependency notation format ?

**Review by zguesmi — APPROVED** [review]
Nice!

File: build.gradle (hunk 2 above)
Comment (james-toussaint) [reply build.gradle]: Bad copy paste, updated thanks :)

File: build.gradle (hunk 1 above)
Comment (james-toussaint) [reply build.gradle]: https://github.com/iExecBlockchainComputing/iexec-common/blob/a5ca5464ae0c5c1a2786ef5a3bacf0845e5d8a09/src/main/java/com/iexec/common/chain/IexecHubAbstractService.java#L26

**Review by mcornaton — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jbern0rd) [reply build.gradle]: Thanks, I missed this usage

**Review by jbern0rd — APPROVED** [review]
Thanks

**Pull request merged**